buildbot/master/files/config/factories.py

```diff
@@ -82,7 +86,10 @@ def make_step(self, command):
         step_env = copy.deepcopy(self.environment)
 
         command = command.split(' ')
-        step_kwargs['command'] = command
+
+        # Add bash -l before every command on Windows builders
+        bash_args = ["bash", "-l"] if is_windows() else []
```

Comment:
i leave ```-c```, to avoid quoting every command, i've tested with appveyor and it work that way too.